lib/buchungsstreber/cli/tui.rb

```diff
@@ -101,6 +101,7 @@ def redraw
 
           @win.setpos(@win.cury, 7)
           @win.attron(Curses::A_BOLD) { @win.addstr("%sh" % e[:time]) }
+          #$stderr.puts [e].inspect
```

Comment:
Someday we should introduce a real logging system....

Reply:
Damnit. >.< This wasn't meant to be committed... I really should configure the debugger.

Reply:
Don't beat yourself up, if you look elsewhere in the code, there are sprinkles of debug statements still left all over. ^_^